packages/docusaurus-theme-classic/src/theme/DocSidebar/Desktop/Content/index.tsx

```diff
@@ -38,9 +39,14 @@ export default function DocSidebarDesktopContent({
   className,
 }: Props): JSX.Element {
   const showAnnouncementBar = useShowAnnouncementBar();
-
+  const ariaLabel = translate({
+    id: 'theme.DocSidebarDesktopContent.sidebarAriaLabel',
+    message: 'Sidebar',
```

Comment:
I agree with "Docs sidebar", but don’t use "navigation" for this aria-label. 

Screen readers will announce the type of role the landmark is. Because of this, you do not need to describe what the landmark is in its label. For example, a declaration of role="navigation" with an of aria-label="Primary navigation" may be announced redundantly as, "primary navigation navigation".

The same for aria-label="Docs pages navigation" in pagination `<nav>`. 

More information about [ARIA: navigation role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role)

Reply:
Thanks, makes sense and it's what I suspected

I guess we should cleanup the other aria labels then ;)

Reply:
Wow, super cool insight Alena!  I can definitely see how hearing navigation twice could be redundant and even confusing to a user. 

Ive adjusted the message for the docs sidebar to `Docs sidebar` as suggested in this thread.